assigner/baserepo.py

```diff
@@ -170,6 +170,25 @@ def already_exists(self):
             return True
         return False
 
+    def get_head(self, branch):
+        if self.repo is None:
+            raise RepoError("No repo to get head from")
```

Comment:
Do we do punctuation on errors?

Reply:
Not that I can see, although we haven't really made a standard for that.